ui/src/views/Tasks/ViewTask/index.jsx

```diff
@@ -107,6 +107,9 @@ const getCachesFromTask = task =>
       artifactsConnection: {
         limit: ARTIFACTS_PAGE_SIZE,
       },
+      dependentsConnection: {
+        limit: 1000,
+      },
```

Comment:
In order to keep the task view minimal, let's show a subset of the dependents at a time (e.g., 5 at a time) with the possibility of changing pages. The `ConnectionDataTable` component might be helpful to use. It renders a paginated table.